homeassistant/components/vallox/sensor.py

```diff
@@ -123,15 +131,16 @@ class ValloxSensorEntityDescription(SensorEntityDescription):
     """Describes Vallox sensor entity."""
 
     metric_key: str | None = None
-    sensor_type: type[ValloxSensor] = ValloxSensor
+    entity_type: type[ValloxSensorEntity] = ValloxSensorEntity
+    round_ndigits: int | None = None
```

Comment:
This could just be a formatting function that defaults to return the value as is. Then we don't need a check in the entity.

Reply:
Can you please elaborate more? For example?

Reply:
We can set a default function as description attribute, eg a lambda that returns the same thing as is input.

This example has a default function set to `round`:
https://github.com/home-assistant/core/blob/3a8748bc9398316b389aa0307fb149829eab20fd/homeassistant/components/airly/sensor.py#L58-L62

Reply:
Interesting. Thank you.